src/pages/iou/request/step/IOURequestStepDate.js

```diff
@@ -47,7 +60,27 @@ function IOURequestStepDate({
      * @param {String} value.moneyRequestCreated
      */
     const updateDate = (value) => {
-        IOU.setMoneyRequestCreated_temporaryForRefactor(transactionID, value.moneyRequestCreated);
+        const newCreated = value.moneyRequestCreated;
+
+        // Only update created if it has changed
+        if (newCreated === currentCreated) {
+            navigateBack();
+            return;
+        }
+
+        // In the split flow, when editing we use SPLIT_TRANSACTION_DRAFT to save draft value
+        if (isEditingSplitBill) {
+            IOU.setDraftSplitTransaction(transaction.transactionID, {created: newCreated});
+            navigateBack();
+            return;
+        }
+
+        IOU.setMoneyRequestCreated(transaction.transactionID, newCreated, action === CONST.IOU.ACTION.CREATE);
+
+        if (action === CONST.IOU.ACTION.EDIT) {
+            IOU.updateMoneyRequestDate(transaction.transactionID, reportID, newCreated);
```

Comment:
https://github.com/Expensify/App/blob/198ca2c503fdd6044ca02f7ae3bfad5d3878a90f/src/pages/EditRequestPage.js#L157-L163

There are new change to this function, please update to add more param